tests/testthat/test_shuffle.r

```diff
@@ -72,3 +72,31 @@ test_that('`seed` generates reproducible intervals',{
    res2 <- bed_shuffle(x, genome, seed = seed)
    expect_identical(res1, res2) 
 })
+
+test_that('`x` intervals not containing all `genome` chroms does not crash', {
+  
+  genome <- tibble::tribble(
+    ~chrom,  ~size,
+    "chr1", 50000000,
+    "chr2", 60000000,
+    "chr3", 80000000
+  ) 
+  
+  
+  x <- tibble::tribble(
+    ~chrom, ~start, ~end,
+    "chr1", 1,    1000,
+    "chr2", 1,    1000,
+    "chr2", 5000, 10000,
+    "chr3", 500,  10000
+  )
+  
+  x_crashes <- tibble::tribble(
+    ~chrom, ~start, ~end,
+    "chr2", 5000, 10000,
+    "chr3", 500,  10000
+  )
+  # crashes 
+  expect_silent(bed_shuffle(x_crashes, genome))
```

Comment:
I'm not sure exactly how to test for this condition without explictly crashing Rstudio, so perhaps it is best left untested. The `incl` test evaluates a different condition, and to be honest I still don't understand how the error was happening in the first place. I'll remove this test for now until I can come up with a more appropriate test condition.